src/main/java/seedu/address/model/person/Telegram.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Optional;
+
+/**
+ * Represents a Person's Telegram handle in UniGenda.
+ * Guarantees: immutable; is valid as declared in
+ */
+public class Telegram {
+    public static final String MESSAGE_CONSTRAINTS = "Telegram username should "
+            + "adhere to the following constraints:\n"
+            + "1. The username should only contain alphanumeric characters and underscore (_).\n"
+            + "2. The username must contain at least 5 characters.";
+    public static final String VALIDATION_REGEX = "^[a-zA-z0-9-_]{5,}$";
+    public static final Telegram DEFAULT_TELEGRAM = new Telegram();
```

Comment:
Rather than calling it DEFAULT_TELEGRAM, perhaps we can call it EMPTY_TELEGRAM to signify that it is empty?

Reply:
okay!